apps/web/components/FrontpageSlider/FrontpageSlider.tsx

```diff
@@ -114,11 +115,19 @@ export const FrontpageSlider: FC<FrontpageSliderProps> = ({
   }
 
   useEffect(() => {
+    const requests = slides.reduce((all, slide) => {
+      if (slide.animationJsonAsset && slide.animationJsonAsset.url) {
+        all.push(axios.get(slide.animationJsonAsset.url))
```

Comment:
Should we use `fetch` instead? Not sure if Axios is already bundled or not, but ideally we don't need it.

Reply:
Axios was already bundled